mmdet/models/dense_heads/base_dense_head.py

```diff
@@ -14,10 +18,266 @@ def loss(self, **kwargs):
         """Compute losses of the head."""
         pass
 
-    @abstractmethod
-    def get_bboxes(self, **kwargs):
-        """Transform network output for a batch into bbox predictions."""
-        pass
+    @force_fp32(apply_to=('cls_scores', 'bbox_preds'))
+    def get_bboxes(self,
+                   cls_scores,
+                   bbox_preds,
+                   score_factors=None,
```

Comment:
Do score_factors need to be force converted to fp32 when running with fp16?

Reply:
Considering it would be `None` in some models, we do not add it to `force_fp32` decorator, but it can be converted to `fp32` automatically when be multiplied to `cls_scores`.